### PR TITLE
Sorted constants in alphabetically order 

### PR DESCRIPTION
**Pull request by sagorika1996**
Addressed to #622

### CODE REVIEW

File: src/main/java/ru/mystamps/web/Url.java [hunk 1]
```diff
@@ -120,29 +120,29 @@ private Url() {
 	public static Map<String, String> asMap(boolean serveContentFromSingleHost) {
 		// There is not all urls but only those which used on views
 		Map<String, String> map = new HashMap<>();
-		map.put("PUBLIC_URL", PUBLIC_URL);
```
Comment (php-coder) [inline src/main/java/ru/mystamps/web/Url.java]: Could you also add a comment here?

File: src/main/java/ru/mystamps/web/support/spring/security/CustomUserDetailsService.java [hunk 2]
```diff
@@ -76,12 +76,12 @@
 		if (userDetails.isAdmin()) {
 			authorities.add(Authority.ADD_COMMENTS_TO_SERIES);
```
Comment (php-coder) [inline src/main/java/ru/mystamps/web/support/spring/security/CustomUserDetailsService.java]: Could you also add a comment here and above?

**Comment by sagorika1996** [comment]
Thanks for the comments! This is my first time trying out GitHub, so I was searching for some trivial bug fixes to get started with and found this.

**Comment by php-coder** [comment]
@sagorika1996 Hi, do you have plans to fix my comments or you aren't interested in anymore?

**Comment by php-coder** [comment]
@sagorika1996 Hi, I don't want to reject your PR but I also can't merge it because it contains 4 commits (instead of 1) and there are couple trivial comments that should be addressed. Please, try to fix them and I'd be happy to accept your contribution!

**Comment by php-coder** [comment]
I'm sorry I can't accept this PR because my comments weren't fixed. If it was a single commit, I would probably merge it. But it has 4 commits for such a trivial change and also I'll need to fix minor issues by myself after that. So I feel that it doesn't worth it.

**Pull request closed without merging**